src/cart/crt_hlc.c

```diff
@@ -96,9 +109,35 @@ int crt_hlc_get_msg(uint64_t msg, uint64_t *hlc_out, uint64_t *offset)
 	return 0;
 }
 
-uint64_t crt_hlc2sec(uint64_t hlc)
+uint64_t crt_hlc2nsec(uint64_t hlc)
 {
-	return (hlc & ~CRT_HLC_MASK) / NSEC_PER_SEC;
+	return hlc / CRT_HLC_NSEC;
+}
+
+uint64_t crt_hlc_from_nsec(uint64_t nsec)
+{
+	return nsec * CRT_HLC_NSEC;
+}
+
+uint64_t crt_hlc2unixnsec(uint64_t hlc)
+{
+	return hlc / CRT_HLC_NSEC + CRT_HLC_START * NSEC_PER_SEC;
+}
+
+int crt_hlc_from_unixnsec(uint64_t unixnsec, uint64_t *hlc)
```

Comment:
same here why not crt_unixnsec2hlc for symmetry?

Reply:
OK.